src/widgets/geo-search/GeoSearchRenderer.js

```diff
@@ -89,9 +89,9 @@ const renderer = (
     mapElement.className = cssClasses.map;
     rootElement.appendChild(mapElement);
 
-    const controlElement = document.createElement('div');
-    controlElement.className = cssClasses.controls;
-    rootElement.appendChild(controlElement);
+    const treeElement = document.createElement('div');
+    treeElement.className = cssClasses.tree;
```

Comment:
It should not have an impact since this class cannot be `undefined` or `null`. But since we use `classnames` everywhere makes sense to use it here also.

Reply:
Exactly @samouss! I will do a last pass for already merged widgets to remove the classnames in the components that don't need `classnames`.